src/test/java/com/codurance/lightaccess/PGOneToManyShould.java

```diff
@@ -0,0 +1,84 @@
+package com.codurance.lightaccess;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameters;
+
+import java.util.*;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.IsEqual.equalTo;
+
+@RunWith(Parameterized.class)
+public class PGOneToManyShould {
+
+
+    private Map<Integer, List<Optional>> actual;
+    private List<Tuple> expected;
+
+    public PGOneToManyShould(Map<Integer, List<Optional>> actual, List<Tuple> expected) {
+        this.actual = actual;
+        this.expected = expected;
+    }
+
+    @Test
+    public void putAnEmptyValueAndNotStoreAnyValue() throws Exception {

```

Comment:
I think the name of this test doesn't describe properly behaviour of these class. For me, it is hard to understand without looking at the code what this class is doing.

Maybe it would be ok to rename it or split it into more tests.